### DISCUSSION TITLE
Working on B2B, we would like to create a new front-end based in Venia

### DISCUSSION THREAD

**Discussion by orienteed** [magento/pwa-studio#3733] [General]
Hello all.

We have worked for multiple B2B customers, extending Venia (and m2) with interesting B2B features like:
- RFQ
- Multiple Saved Carts
- Re-Orders
- CSR
- Import CSV as new cart
- Edit and export Cart as PDF
- PDP for B2B
- Agent flow
- and so on ...

We would like to share our code.
Our idea is to make a fork of Venia and create a new open source project totally focused on B2B.

Do you see any issue? what do you think?

thanks for your opinion
regards!

**Comment by briscoda** [comment]
I would be interested in seeing and contributing to B2B features for Venia. I see it is on the roadmap for this year, but it would be helpful.

**Reply by validoai** [reply]
Hello @briscoda! Our point of view is that Venia is evolving fast and from the technology point of view seems to make a step forward to any new release (also looking at the roadmap).

Our idea is to keep following the Venia evolution but focusing just on B2B features (still Venia is too tight to the B2C model).

During next days we will publish the repo. Any help is really welcome. 

We hope to contribute to the M2 community on the B2B side :-)

**Reply by briscoda** [reply]
Awesome! I am extremely interested in collaboration on this front.

**Reply by orienteed** [reply]
Hello @briscoda and all
We have just released our first version: https://github.com/orienteed/b2bstore
demo: https://www.b2bstore.io/demo

We look forward to receive your feedback!
regards
